src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -1386,6 +1387,31 @@ internal override IEnumerable<FieldSymbol> GetFieldsToEmit()
                             yield return associatedField;
                         }
                         break;
+                    case SymbolKind.Property:
+                        if (m is SourcePropertySymbolBase propertySymbol)
+                        {
+                            // Ensure the binding is done so we guarantee that we have the BackingField set if necessary.
+                            if (propertySymbol.BackingField is null &&
+                                propertySymbol.CSharpSyntaxNode is BasePropertyDeclarationSyntax propertySyntax)
+                            {
+                                if (propertySymbol.GetMethod is SourceMemberMethodSymbol getMethod)
+                                {
+                                    getMethod.TryGetBodyBinder()?.BindMethodBody(getMethod.SyntaxNode, BindingDiagnosticBag.Discarded);
```

Comment:
>getMethod.TryGetBodyBinder()?.BindMethodBody(getMethod.SyntaxNode, BindingDiagnosticBag.Discarded); [](http://example.com/codeflow?start=36&length=99)

We probably don't want to go through all this trouble if there are no `field` keywords in the body. We might want to capture the fact of presence of the keyword when we create symbol(s), syntax is readily available at the time.

Reply:
I'm not sure if the way I fixed this is good enough.